src/panels/SkyBrowserPanel/WorldWideTelescope/WwtProvider/types.d.ts

```diff
@@ -0,0 +1,60 @@
+interface HideChromeMessage {
+  event: 'modify_settings';
+  settings: [['hideAllChrome', true]];
+  target: 'app';
+}
+
+interface LoadCollectionMessage {
+  event: 'load_image_collection';
+  url: string;
+  loadChildFolders: true;
+}
+
+interface BorderRadiusMessage {
+  event: 'set_border_radius';
+  data: number;
```

Comment:
Add a comment explainng that this is the radius value and what type of value is expected